tests/integration_tests/allennlp_tests/test_allennlp.py

```diff
@@ -197,6 +198,7 @@ def test_allennlp_executor_with_options() -> None:
             executor.run()
             assert mock_obj.call_args[1]["force"]
             assert mock_obj.call_args[1]["file_friendly_logging"]
+            assert mock_obj.call_args[1]["include_package"] == [package_name]
```

Comment:
If `include_package` is `str`, it is converted to `list` in `AllenNLPExecutor`.
https://github.com/optuna/optuna/blob/master/optuna/integration/allennlp.py#L295-L300